src/code/z_horse.c

```diff
@@ -14,15 +23,23 @@ s32 func_8006CFC0(s32 scene) {
     return 0;
 }
 
-void func_8006D074(GlobalContext* globalCtx) {
+void Horse_SetSpawnInHyruleField(GlobalContext* globalCtx) {
```

Comment:
I'm not sure the function should mention the scene

Based on usage and debug strings it looks more like a fallback / set default kind of function